src/shared/components/post/post-listing.tsx

```diff
@@ -1723,6 +1725,20 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
     event.preventDefault();
     i.setState({ imageExpanded: !i.state.imageExpanded });
     setupTippy();
+
+    const auth = myAuth();
+    if (
+      auth &&
+      (typeof i.props.post_view.read === "undefined" ||
+        i.props.post_view.read === false)
```

Comment:
Is the check on the type necessary? If the type of `post_view.read` is `boolean | undefined`, `!i.props.post_view.read` should suffice.

Reply:
Thank you for the feedback. I have fixed it.